SdsTsDotNet/Program.cs

```diff
@@ -87,7 +87,7 @@ public static async Task<bool> MainAsync(bool test = false)
                 };
                 pressure_stream = await metadataService.GetOrCreateStreamAsync(pressure_stream).ConfigureAwait(false);
 
```

Comment:
var missed on line 82

Reply:
good catch, I missed the new () statement being simplified in his email. I didn't know that was even a thing. They should all be changed now

Reply:
yeah, i have mixed feeling on the new simplification.  I think it is cleaner and neat overall, but i haven't seen it that much so it feels odd to me still